packages/common/src/ui/components/inputs/Autocomplete/AutocompleteList.tsx

```diff
@@ -17,12 +17,16 @@ export type AutocompleteListProps<T> = {
   loading?: boolean;
   loadingText?: React.ReactNode;
   noOptionsText?: React.ReactNode;
-  onChange?: AutocompleteOnChange<T>;
+  onChange?: AutocompleteOnChange<T | T[]>;
```

Comment:
this is because I've added `multiple` as an optional prop and the return shape is now an item or an array